ge03_team4/models/product_template.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since python version 3, the standard python encoding is utf-8 so it is no longer necessary to add it :)
Also, why are you using it only in this file? you have 3 pythin files and only use in this one, always think in consistency.